recipe/meta.yaml

```diff
@@ -37,16 +38,27 @@ requirements:
 test:
   imports:
     - evidently
-  commands:
-    - pip check
+  source_files:
+    - tests/*.py
```

Comment:
is this correct? there are subdirs with more tests in `tests/`, these tests are not required?

Reply:
Thanks. Fixed it. Trying to rebuild with all tests

Reply:
As it fails now I'll debug it tomorrow. Some packages are missing for the test environment (`ipython`, etc.)

Reply:
Done!